packages/shared/src/components/PostsSearch.tsx

```diff
@@ -122,7 +122,7 @@ export default function PostsSearch({
       <SearchField
         className="flex-1 compact"
         inputId="posts-search"
-        compact
+        fieldSize="medium"
```

Comment:
I see now that it's correct for the "feed", but could you check with Tsahi that he intended the feeds to have the "medium" search

Reply:
I think the size is really necessary as it came from the guideline.

![Screen Shot 2022-03-22 at 4 51 58 PM](https://user-images.githubusercontent.com/13744167/159442859-085843ee-f6d6-47df-9be1-1ac53f93c01f.png)

Reply:
Yes it is needed

My question is to ask Tsahi if "medium" is the right size for the feeds 🙌

Reply:
Oh, sorry for the misunderstanding. Let me ping him up. Thank you!